R/utils_render_html.R

```diff
@@ -1072,6 +1093,21 @@ create_footnotes_component_h <- function(data) {
     dplyr::select(fs_id, footnotes) %>%
     dplyr::distinct()
 
+  # Get the style attrs for the footnotes
+  if ("footnotes" %in% styles_tbl$locname) {
+    footnotes_style <- dplyr::filter(styles_tbl, locname == "footnotes")
+
+    footnotes_styles <-
+      if (nrow(footnotes_style) > 0) {
+        footnotes_style$html_style
+      } else {
+        NULL
+      }
+
+  } else {
+    footnotes_styles <- NA_character_
```

Comment:
should be NULL

Reply:
Now changed. Checked the code elsewhere in the file and `NA_character_` was only used in this PR (i.e., `NULL` is used consistently elsewhere).